lte/gateway/python/integ_tests/s1aptests/s1ap_wrapper.py

```diff
@@ -81,6 +81,12 @@ def __init__(
         self._s1_util = S1ApUtil()
         self._enBConfig(ip_version)
 
+        if os.environ.get("FED_MODE") == "True":
```

Comment:
nit: Please call the env variable FEDERATED_MODE so it matches with the python var. If you want to use one.